modules/ELT/pages/getting-started-with-airbyte.adoc

```diff
@@ -0,0 +1,197 @@
+= Airbyte with Teradata Vantage
+:experimental:
+:page-author: Krutik Pathak
+:page-email: krutik.pathak@teradata.com
+:page-revdate: June 1st, 2023
+:description: Use Airbyte with Teradata Vantage.
+:keywords: airbyte, data warehouses, compute storage separation, teradata, vantage, cloud data platform, object storage, data transfer, data extraction, enterprise analytics, elt.
+:tabs:
+:dir: getting-started-with-airbyte
+
+== Overview
+
+This tutorial showcases how to use Airbyte (an Open-source Extract Load Transform Tool) with Teradata Vantage. We work with a very simple end-to-end setup to load the employees’ data from Google Sheets to Teradata Vantage using Airbyte.
```

Comment:
Some words here are capitalized while they should not be. For more info https://langster.org/en/blog/capitalization-rules-in-english-the-basics
```suggestion
This tutorial showcases how to use Airbyte (an open-source Extract Load Transform tool) with Teradata Vantage. We work with a very simple end-to-end setup to load the employees’ data from Google Sheets to Teradata Vantage using Airbyte.
```